util/cron/common-slurm-gasnet.bash

```diff
@@ -1,26 +1,27 @@
 #
 # Configure environment for
```

Comment:
I'm thinking this should be called common-slurm-gasnet-cray-cs or something to note that's it's pretty cray-cs specific (at least since it's sourcing common-cray-cs)